docs/index.md

```diff
@@ -15,43 +15,43 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
-| visit.date     | Date     | yyyy-mm-dd                                                 |                                                                                                                                                                                                                             |
-| visit.location | Text     |                                                            | Text Abbreviation for facility of visit location                                                                                                                                                                            |
-| visit.provider | Text     |                                                            | Full provider name, last name, or just NPI                                                                                                                                                                                  |
-| Lab Array      |          |                                                            |                                                                                                                                                                                                                             |
-| lab.lab_id     | Numeric  | 16 Digit GE ID                                             | Unique GE ID for specific lab result (ObsID)                                                                                                                                                                                |
-| lab.type       | Text     | BLL                                                        | Static BLL unless we identify additional labs to include                                                                                                                                                                    |
+| Field          | Format   | Constraints                                                | Notes/Questions                                                                                                   |
+|----------------|----------|------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| timestamp      | TimeDate | yyyy-mm-dd hh:mm:ss.sss-hh:mm                              | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                                                |
+| clinic_id      | Text     |                                                            | Two Characted Organization ID (NN- NearNorth  EF- ErieFamily)                                                     |
+| location_id    | Text     |                                                            | Specific Location Abbreviation in Centricity                                                                      |
```

Comment:
@RaedMan is going to see if other networks also have a `clinic_id` and `location_id` equivalents. Namely, whether they have a concept of a clinic within their networks (e.g., a distinct sub-entity or sub-group within the network) and whether they also have a distinct location ID .